packaging/graphql-engine/Dockerfile

```
@@ -9,4 +9,5 @@ COPY rootfs/srv/ /srv/
 
 HEALTHCHECK --start-period=10s CMD curl -f http://localhost:8080/healthz || exit 1
 
-CMD "${HGE_BINARY}" serve
+ENTRYPOINT [ "${HGE_BINARY}" ]
+CMD [ "serve" ]
```

This is a breaking change for any scripts orchestrating a HGE Docker container. For example, they might provide their own entrypoint but use the command syntax rather than `--entrypoint`.

I suggest we keep both the command and the argument in the `CMD` directive.

```suggestion
CMD ["${HGE_BINARY}", "serve"]
```

That said, this is moot until my other question is answered.

---

Right; happy to revert to this after we choose a direction for above.